.travis.yml

```diff
@@ -15,7 +15,11 @@ before_install:
 install: sudo make install_pip
 
 script:
+  - sudo wget https://github.com/openshift/origin/releases/download/v3.7.0/openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz
+  - sudo tar -xzvf ./openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz
+  - sudo ln -s ./openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit/oc /home/travis/oc
```

Comment:
why not adding the symlink to $PATH directory?

Reply:
thx, I did it, but `oc` is not visible for some reason.

```
$ sudo make -C examples/testing-module check-openshift-memcached
make: Entering directory `/home/travis/build/fedora-modularity/meta-test-family/examples/testing-module'
sudo oc cluster up
sudo: oc: command not found
make: *** [check-openshift-memcached] Error 1
make: Leaving directory `/home/travis/build/fedora-modularity/meta-test-family/examples/testing-module'
```